packages/cli/src/commands/contract/test.ts

```diff
@@ -40,32 +40,18 @@ export class TestContract extends Command {
     await ensureSwankyProject();
     const config = await getSwankyConfig();
 
-    const contractNames = [];
-    if (flags.all) {
-      for (const contractName of Object.keys(config.contracts)) {
-        console.log(`${contractName} contract is found`);
-        contractNames.push(contractName);
-      }
-    } else {
-      contractNames.push(args.contractName);
-    }
+    const contractNames = flags.all ? Object.keys(config.contracts) : args.contractName;
 
-    const projectDir = path.resolve();
-    const testDir = path.resolve("test");
+    const testDir = path.resolve("tests");
```

Comment:
If you rename this to `tests` then other parts depending on this path name should be updated as well. [here](https://github.com/AstarNetwork/swanky-cli/pull/136/files#diff-058bea17199616b199bc211ad0a539af68ef7995a978f571dcd159c4a46c7dd8R149) & [here](https://github.com/AstarNetwork/swanky-cli/pull/136/files#diff-9fed1ecd47dbad64451c84eb0824445726888037915e89c0c96a325c48745a49R71) should be updated as well. otherwise, everything broken.